### PR TITLE
Expose granular mixins

### PR DESCRIPTION
**Pull request by blasten**
`--paper-font-common-base`
` --paper-font-common-code`
`--paper-font-common-expensive-kerning`
`--paper-font-common-nowrap`

cc @cdata @notwaldorf

### CODE REVIEW

File: typography.html [hunk 1]
```diff
@@ -18,10 +18,6 @@
 
     /* Shared Styles */
 
-    /*
```
Comment (notwaldorf) [inline typography.html]: If you are removing this comment, then you should probably remove the commented out `@applies` for these styles in the mixins in this file.

Comment (notwaldorf) [inline typography.html]: Or leave a note around the commented out `@applies` as to why they're commented out? Basically, I don't want this comment to be lost, if @morethanreal left it in here for a reason.

Comment (blasten) [inline typography.html]: @notwaldorf there you go. I added the comment back.

**Comment by morethanreal** [comment]
lgtm

**Pull request merged**